packages/protocol/contracts/layer1/tiers/TierProviderBase.sol

```diff
@@ -64,22 +63,22 @@ abstract contract TierProviderBase is ITierProvider {
         pure
         returns (ITierProvider.Tier memory)
     {
-        // cooldownWindow is 4 hours and provingWindow is 60 minutes
-        return _buildTier(_verifierName, BOND_UNIT * 2, 4, 60);
+        // cooldownWindow is 1440 minutes and provingWindow is 60 minutes
```

Comment:
Seems that the proving window is 5 hrs (`240mins + 60mins`) instead of `60mins`?

Reply:
Yes, lets change it later with an announcement.

Reply:
But before ontake update, i think on mainnet its `60mins`